aleph-client/src/lib.rs

```diff
@@ -68,8 +78,20 @@ pub use connections::{
 };
 
 /// An alias for a configuration of live chain, e.g. block index type, hash type.
-type AlephConfig = PolkadotConfig;
-type ParamsBuilder = subxt::tx::PolkadotExtrinsicParamsBuilder<SubstrateConfig>;
+pub enum AlephConfig {}
+
+impl Config for AlephConfig {
```

Comment:
what is the difference now between `AlephConfig` and `PolkadotConfig`? in `AccountId`?

Reply:
Good point, it's AccountId, Address and Signature - refactored to emphasise that